hbase-examples/src/main/java/org/apache/hadoop/hbase/thrift2/DemoClient.java

```diff
@@ -90,8 +93,13 @@ public Void run() throws Exception {
   public void run() throws Exception {
     int timeout = 10000;
     boolean framed = false;
+    TTransport transport = null;
+    try {
+      transport = new TSocket(new TConfiguration(), host, port, timeout);
+    } catch (TTransportException e) {
```

Comment:
Why we need a try catch here? The method is declared as throwing Exception...

Reply:
Yeah, no need of try/catch here.